Sources/Build/BuildPlan.swift

```diff
@@ -629,9 +629,9 @@ public class BuildPlan: SPMBuildCore.BuildPlan {
         // Note: This will come from build settings in future.
         for target in dependencies.staticTargets {
             if case let target as ClangTarget = target.underlyingTarget, target.isCXX {
-                if buildParameters.hostTriple.isDarwin() {
+                if buildParameters.triple.isDarwin() {
```

Comment:
unrelated to this PR: is `triple` the target / destination triple? should we rename it accordingly to make the code more obvious?